config/application.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'boot'
+require 'uri'
```

Comment:
I think this might be a remnant from the error you were experiencing before removing your Gemfile.lock. If not, I'm curious about why you added it.